packages/admin-panel/src/VizBuilderApp/api/mutations/useExportVisualisation.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { useMutation } from 'react-query';
+import { download } from '../api';
+
+export const useExportVisualisation = config =>
+  useMutation(['export/dashboardVisualisation', config], () =>
```

Comment:
This method is called `useExportVisualisation`, but specifically calls `export/dashboardVisualisation`. We can refactor this when we add map overlay support, either by providing the type of viz to export (if the process is almost identical), or by renaming it to `useExportDashboardVisualisation`.

The decision was made based on an existing mutation (`useSaveVisualisation`)